main_logic/tts_client.py

```
@@ -1959,452 +2139,159 @@ def minimax_tts_worker(request_queue, response_queue, audio_api_key, voice_id, b
 
-                if not await _send_text_chunk(tts_text):
-                    continue
+                    # 累积文本并尝试按句切分合成
+                    if tts_text and tts_text.strip():
+                        text_buffer += tts_text
+                        _try_split_and_synth(client, current_speech_id)
 
-        except Exception as e:
-            _enqueue_error(response_queue, f"MiniMax TTS Worker 错误: {e}")
+                await _cancel_all_synth()
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)